### PR TITLE
Iss hipcms 89

### PR DESCRIPTION
**Pull request by maltehol**

### CODE REVIEW

**Review by janis-kra — COMMENTED** [review]
Lots of eslint errors, I'll fix it with `eslint app.js --fix`

File: bin/www [hunk 1]
```diff
@@ -1,10 +1,9 @@
-﻿#!/usr/bin/env node
-var debug = require("debug")("OnlineEditorsExampleNodeJS");
-var app = require("../app");
+﻿var debug = console.log;
+var app = require('../app-compiled');
```
Comment (janis-kra) [inline bin/www]: app-compiled does not exist and is created nowhere - I guess you forgot to commit some kind of build script.

Why would we need to compile anyway? I dont think there is any fancy ES7 stuff that we use, so node v6 should be able to run it all

File: permissions.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+const request = require('request');
+const config = require('config');
+const apiUrl = config.get('server').get('cmsApiUrl');
+
+function callback(error, response, process) {
+  if (!error && response.statusCode == 200) {
```
Comment (janis-kra) [inline permissions.js]: always use `===` in javascript

File: permissions.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    process(true);
+    return;
+  }
+  if (!error && response.statusCode == 401) {
```
Comment (janis-kra) [inline permissions.js]: see above

File: permissions.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    process(false);
+    return;
+  }
+  process(error);
+}
+
+var permissions = {};
```
Comment (janis-kra) [inline permissions.js]: always use `const` if possible

**Review by janis-kra — CHANGES_REQUESTED** [review]

File: app.js [hunk 5]
```diff
@@ -40,28 +40,32 @@ const jws = require('jws-jwk');
...
   return ret;
 };
-String.prototype.format = function () {
+String.prototype.format = function format() {
   let text = this.toString();
 
   if (!arguments.length) return text;
 
-  for (let i = 0; i < arguments.length; i++) {
+  for (let i = 0; i < arguments.length; i + 1) {
```
Comment (maltehol) [inline app.js:68]: Why not i++?

Comment (janis-kra) [reply app.js:68]: Our eslint config wants it that way (see http://eslint.org/docs/rules/no-plusplus).
We can of course just change the config, not a big deal.

Comment (janis-kra) [reply app.js:68]: I'll merge this now, feel free to change the config separately if you want.

**Review by janis-kra — APPROVED** [review]

**Pull request merged**